.github/workflows/release.yaml

```diff
@@ -0,0 +1,131 @@
+on:
+  push:
+    tags:
+      - 'litani-*'
+
+name: Create Release and package
+
+jobs:
+  get-version-information:
+    name: Get Version Information
+    runs-on: ubuntu-20.04
+    outputs:
+      version: ${{ steps.split-version.outputs._1 }}
+    steps:
+      - uses: jungwinter/split@v1
```

Comment:
Why not v2?

I guess for each of these externally-authored steps that we use, we should audit them and then stick to a particular version, right? What about security issues?

Reply:
We use this one for a very simple change and I guess version won't matter here but we can definitely keep it updated.
Also I saw this being used in one of the `cbmc` workflows for the same.

Reply:
What about:
> I guess for each of these externally-authored steps that we use, we should audit them and then stick to a particular version, right? What about security issues?